ProcessLib/SmallDeformationWithLIE/Common/FractureProperty.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
```

Comment:
There is https://github.com/ufz/ogs/issues/1376 and so far only two people liked it...